superset-frontend/src/features/alerts/components/AlertReportCronScheduler.tsx

```diff
@@ -51,17 +51,15 @@ export const AlertReportCronScheduler: FC<AlertReportCronSchedulerProps> = ({
   onChange,
 }) => {
   const theme = useTheme();
-  const inputRef = useRef<AntdInput>(null);
```

Comment:
Is this not necessary after the upgrade?

Reply:
setValue is obsolete and redundant on the customSetValue callback and i think handlePressEnter can use value directly so this becomes unneccessary.